src/main/kotlin/com/learnspigot/bot/command/ToggleCommand.kt

```diff
@@ -22,7 +22,7 @@ class ToggleCommand(private val guild: Guild, private val bot: JDA, private val
                 datastore.save(profile)
                 it.replyEmbed({
```

Comment:
Can I ask you to make this ephemeral aswell?

Reply:
Should be added